portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,45 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.Random;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+        String[] randomFacts = new String[]{
+            "I like to do voice acting. Currently, I can do 12ish different voices, which are from the characters of La Commedia dell'Arte",
+            "I like to compose loops for videogames and for theatre plays. I have composed the soundtrack for two plays from my country and a song for a college play.",
+            "A professor from my college known as Davalos allowed us to create memes for 3 extra points in the course. " +
+            "\nI'm very bad creating memes, so in our Computer Organization course I asked  him if I could develop a videogame - meme for the extra points. That's how Davatrix was born.\n" +
+            "One year later, I took the Network Fundamentals course with him again, so I created the sequel of Davatrix, in which he has to help Vinton Cerf to save the internet.",
+        };
+        Random rand = new Random();
```

Comment:
This will work, but also creates a new string array and Random variable every time doGet is called. It's probably better practice to create them once in the class and reuse them each time in the doGet function.

Reply:
Thank you! I will add them as attributes of the class.